media/video/__init__.py

```diff
@@ -0,0 +1,80 @@
+"""Upload module"""
+import logging
+from bson import ObjectId
+from flask import request
+import json
+from media import get_collection, blueprint
+from flask import Blueprint
+
+bp = Blueprint('project', __name__)
+
+
+class JSONEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, ObjectId):
+            return str(o)
+        return json.JSONEncoder.default(self, o)
+
+
+@bp.route('/project', methods=['POST'])
+def create_video_editor():
+    if request.method == 'POST':
+        files = request.files.to_dict(flat=False)['media']
+        user_agent = request.headers.environ['HTTP_USER_AGENT']
+
+        return create_video(files, user_agent)
+
+
+@bp.route('/project/<path:video_id>', methods=['GET', 'PUT', 'DELETE'])
+def process_video_editor(video_id):
+    """Keep previous url for backward compatibility"""
+    if request.method == 'GET':
+        return get_video(video_id)
```

Comment:
We must return http response here not iter/list. Implement this in next prs, because this one is just initial structure.